pkg/apis/virt/v1alpha1/types.go

```diff
@@ -95,6 +110,7 @@ type InterfaceBindingMethod struct {
 	Bridge     *InterfaceBridge     `json:"bridge,omitempty"`
 	Masquerade *InterfaceMasquerade `json:"masquerade,omitempty"`
 	SRIOV      *InterfaceSRIOV      `json:"sriov,omitempty"`
+	Vhostuser  *InterfaceVhostuser  `json:"vhostuser,omitempty"`
```

Comment:
-> VhostUser?

Reply:
It's named `vhostuser` in both Kubevirt and Kube-OVN.